packages/react-components/react-dialog/package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@griffel/react": "^1.3.0",
     "@fluentui/react-utilities": "^9.1.0",
+    "@fluentui/react-trigger": "^9.0.0-alpha.0",
```

Comment:
this should not be an alpha package, especially since we moved it out of a stable package